tests/EdFi.Ods.Api.IntegrationTests/PersonType.cs

```diff
@@ -0,0 +1,14 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.Ods.Api.IntegrationTests
+{
+    public enum PersonType
```

Comment:
This is probably a good idea, but going to look weird if Student/Parent aren't using it.